portal/models/user.py

```diff
@@ -1025,14 +1030,13 @@ def merge_with(self, other_id):
             abort(404, 'other_id {} not found'.format(other_id))
 
         # direct attributes on user
-        # intentionally skip {email, reset_password_token}
-        for attr in (
-            'password', 'first_name', 'last_name', 'birthdate', 'phone_id',
-            'alt_phone_id', 'gender', 'locale_id', 'timezone', 'confirmed_at',
-            'registered', 'image_url', 'active', 'deleted_id', 'deceased_id'):
-            if not getattr(other, attr):
-                continue
-            setattr(self, attr, getattr(other, attr))
+        # intentionally skip {id, email, reset_password_token}
+        exclude = ['id', 'email', 'reset_password_token']
```

Comment:
Well done - I like the column_names method.

This will work, but there's a generator pattern you may like, that I think improves legibility:
`for attr in (col for col in self.column_names() if col not in exclude):`